### PR TITLE
Bump bundle to use channels for 5.3

### PR DESCRIPTION
**Pull request by periklis**
### Description
This PR is preparation work for release management of the upcoming `release-5.3`. This changes the bundle annotations to reflect the channels based on the new product version namely `5.3`.

To address: https://issues.redhat.com/browse/LOG-1749

Depends on: 
- https://github.com/openshift/release/pull/21885
- https://github.com/openshift/release/pull/21933
- https://github.com/openshift/release/pull/21983

/cc @igor-karpukhin

### CODE REVIEW

File: olm_deploy/operatorregistry/elasticsearch-operator.package.yaml [hunk 1]
```diff
@@ -1,9 +1,9 @@
 packageName: elasticsearch-operator
 defaultChannel: "stable"
 channels:
-- name: "5.2"
-  currentCSV: elasticsearch-operator.v5.2.0
+- name: "5.3"
+  currentCSV: elasticsearch-operator.v5.3.0
 - name: "stable"
-  currentCSV: elasticsearch-operator.v5.2.0
+  currentCSV: elasticsearch-operator.v5.3.0
```
Comment (jcantrill) [inline olm_deploy/operatorregistry/elasticsearch-operator.package.yaml:7]: I don't recall if this `stable-5.2` needs to be modified too.

**Review by jcantrill — COMMENTED** [review]
/approve

File: olm_deploy/operatorregistry/elasticsearch-operator.package.yaml (hunk 1 above)
Comment (periklis) [reply olm_deploy/operatorregistry/elasticsearch-operator.package.yaml:7]: Technically you are right, it should be `stable-5.3`. However this the package.yaml is used only for olm-deploy. Production deployments are fine as long as the annotations.yaml says:
```
  operators.operatorframework.io.bundle.channels.v1: stable,stable-5.3
```

**Review by sasagarw — APPROVED** [review]
/lgtm

**Pull request merged**